Sources/StreamVideoSwiftUI/CallView/VideoParticipantsView.swift

```diff
@@ -380,15 +381,23 @@ public struct VideoCallParticipantView<Factory: ViewFactory>: View {
     }
 
     private var showVideo: Bool {
-        participant.shouldDisplayTrack || customData["videoOn"]?.boolValue == true
+        if isLocalParticipant {
```

Comment:
This is actually improving the logic on showing the placeholder (user avatar) when video is off for localparticipant.

Reply:
did we merge the customData change? Wasn't this handled there?